File: 241108/청소당번 정하기/cleaning-numbering.py
n = int(input())

cnt1 = 0
cnt2 = 0
cnt3 = 0

for i in range(1, n+1):
    if i % 12 == 0:
        cnt3 += 1
    elif i % 3 == 0:
        cnt2 += 1
    elif i % 2 == 0:
        cnt1 += 1
        
print(cnt1, cnt2, cnt3)

File: 241108/청소당번 정하기/cleaning-numbering.c
#include <stdio.h>

int main() {
    int n;
    int cnt2 = 0, cnt3 = 0, cnt12 = 0;
    scanf("%d", &n);

    for(int i = 1; i <= n; i++){
        if (i % 12 == 0)
            cnt12++;
        else if (i % 3 == 0)
            cnt3++;
        else if (i % 2 == 0)
            cnt2++;
    }
    printf("%d %d %d", cnt2, cnt3, cnt12);
    return 0;
}